idarling/core/events.py

```diff
@@ -297,7 +300,12 @@ def __call__(self):
             py_type = py_type[1:]
         if len(py_type) >= 2:
             if self.name:
-                self.ea = ida_struct.get_member_by_fullname(self.name).id
+                r = ida_struct.get_member_by_fullname(self.name)
+                if type(r)is list:
+                    assert type(r[0]) is ida_struct.member_t
```

Comment:
I'd like to avoid having an assert in the server code, as it will exit the server if that is `False`. Having a test instead and returning cleanly without applying the event seems better

Reply:
Removed).
Need add logger to events. Do you have some ideas how?

Reply:
Good idea about adding logger to events.py. Tracked in https://github.com/fidgetingbits/IDArling/issues/60